docs/guide/topics/error_handling.rst

```diff
@@ -0,0 +1,361 @@
+.. currentmodule:: discord
+
+
+==========================
+Errors and Error Handling
+==========================
+
+Handling Errors
+===============
+
+When an error ocurrs in a command or an event, it will, by default, show an error in the console to denote that the error 
+ocurred and has been silently ignored. In order to avoid this, an error handler is needed.
```

Comment:
```suggestion
ocurred and has been silently handled. In order to change this behaviour, an error handler is required.
```

Slightly change the wording here to read a little better and more human-like. Also less "absolute" in their definitions.